src/styles/partials/components/_site-header.scss

```diff
@@ -0,0 +1,14 @@
+.header {
```

Comment:
just a reminder this is gonig to be `.dg_header`

Reply:
fixed